js/client-chat-tournament.js

```diff
@@ -318,7 +318,7 @@
 					if (data[0] === 'off') {
 						this.room.$chat.append("<div class=\"notice tournament-message-autodq-off\">The tournament's automatic disqualify timer has been turned off.</div>");
 					} else if (data[0] === 'on') {
-						this.room.$chat.append("<div class=\"notice tournament-message-autodq-off\">The tournament's automatic disqualify timer has been set to " + (data[1] / 1000 / 60) + " minutes.</div>");
+						this.room.$chat.append("<div class=\"notice tournament-message-autodq-off\">The tournament's automatic disqualify timer has been set to " + (data[1] / 1000 / 60) + " minute" + (data[1] === 60000 ? "" : "s") + ".</div>");
```

Comment:
I'd say create a temporary variable for `(data[1] / 1000 / 60)` to make this read better.